{{ cookiecutter.repo_name }}/.devcontainer/README.md

```diff
@@ -13,4 +11,4 @@ View menu -> Command Palette -> DevContainers: Reopen in Container.
 Other requirements:
 1. An environment file (env.yaml) is required placed in the .devcontainer folder of the  
```

Comment:
Line 12 is tough to read. Looks like it should read something like _"An environment file (env.yaml) is required in the .devcontainer ..."_

Also, this sentence is very "wordy" - everything after _"folder of the project"_ can be removed

- [ ] Fix sentence
- [ ] Remove excess words